doc/ingesting.md

```diff
@@ -253,8 +253,7 @@ What happens when you run `bin/ingest -f etd /path/to/etds/etdadmin_upload*`, th
         '/tmp/etdadmin_upload_56186.zip/NAME_ucsb_0035D_67/cat.gif'
       ]
     }
-    ```
-
+    ```Ω
```

Comment:
Is `Ω` a typo?

Reply:
Good catch! I get the  👑 for typos. Will make the omega go away!